### ISSUE TITLE
Workflow properties missing in latest release

### ISSUE THREAD

**Issue by innovate-invent** [galaxyproject/bioblend#364]
I had a script accessing the 'owner' property of a workflow as returned by `conn.workflows.list(published=True)`.
Recently I updated my python environment and now the 'owner' property is missing along with a number of others.

**Comment by nsoranzo** [comment]
@innovate-invent The `owner` property was never exposed as an attribute of the `Workflow` objects returned by [gi.workflows.list()](https://bioblend.readthedocs.io/en/latest/api_docs/galaxy/all.html#bioblend.galaxy.objects.client.ObjWorkflowClient.list). You should be able to extract it with `<Workflow_Object>.wrapped['owner']` or by adding `'owner'` [here](https://github.com/galaxyproject/bioblend/blob/master/bioblend/galaxy/objects/wrappers.py#L187). If the latter works, a PR is welcome!

**Comment by innovate-invent** [comment]
I am not sure what allows [this script](https://github.com/brinkmanlab/islandcompare-cli/blob/00aaeaa288a9e79002acc5674b9471a1daae0fe3/islandcompare.py#L157) to function with 0.13.0 but it errors when trying to access the 'owner' property in 0.14.0

**Comment by nsoranzo** [comment]
> Woops, looks like a monkey patch I was using allowed it to work.

That's what I suspected! Any way, should be fixed by commit 7ce14b7bada7e6f4e8a91a04297ab42413c65f6a .

**Comment by innovate-invent** [comment]
If it is worth anything to you, these are my patches:
```
def get_invocations(self, workflow_id, history_id=None, user_id=None, include_terminal=True, limit=None, view='collection', step_details=False):
    url = self._invocations_url(workflow_id)
    params = {'include_terminal': include_terminal, 'view': view, 'step_details': step_details}
    if history_id: params['history_id'] = history_id
    if user_id: params['user_id'] = user_id
    if limit: params['limit'] = limit
    return self._get(url=url, params=params)


WorkflowClient.get_invocations = get_invocations
Workflow.BASE_ATTRS += ('owner', 'number_of_steps', 'show_in_tool_panel', 'latest_workflow_uuid')
```
